Sources/SwiftProtobuf/Message.swift

```diff
@@ -90,16 +94,12 @@ public protocol Message: CustomDebugStringConvertible {
                                                extensions: ExtensionSet?) throws
 
   //
-  // Protobuf Text decoding
-  //
-  mutating func decodeTextFormat(from: inout TextFormatDecoder) throws
-
-  //
   // google.protobuf.Any support
   //
 
-  // Decode from an `Any` (which might itself have been decoded from JSON,
-  // protobuf, or another `Any`).
+  /// Decode this object from an `Any` (which might itself have been
+  /// decoded from JSON, protobuf, or another `Any`).  This is the
+  /// analog of `unpack` in the Google C++ or Java API.
   init(any: Google_Protobuf_Any) throws
```

Comment:
I'm down with `init(unpackingAny:)`. 👍 

I don't think we need to worry about confusion with Swift's built-in `Any` then, since the verb and argument type will make it clear.